Makefile

```diff
@@ -289,26 +317,58 @@ git-diff-check:
 
 RPM_SPECFILE=rpmbuild/SPECS/osbuild-$(COMMIT).spec
 RPM_TARBALL=rpmbuild/SOURCES/osbuild-$(COMMIT).tar.gz
+RPM_SPECFILE_DEVEL=rpmbuild/SPECS/osbuild-SRC.spec
+RPM_TARBALL_DEVEL=rpmbuild/SOURCES/osbuild-SRC.tar.gz
 
 $(RPM_SPECFILE):
 	mkdir -p $(CURDIR)/rpmbuild/SPECS
 	(echo "%global commit $(COMMIT)"; git show HEAD:osbuild.spec) > $(RPM_SPECFILE)
 
+$(RPM_SPECFILE_DEVEL): osbuild.spec
+	mkdir -p $(CURDIR)/rpmbuild/SPECS
+	echo "%global commit SRC" > $@
+	echo "%define _rpmfilename %%{NAME}.rpm" >> $@
```

Comment:
Why is this needed, AIUI we build the rpms in the devcontainer (which will be fresh?) and then we do `RUN rpm -i /app/rpmbuild/RPMS/*.rpm` this would work with the default naming too, or am I missing something?

Reply:
Here also just making it obvious noone mixes this up with something to be released